validator/README.md

```diff
@@ -0,0 +1,76 @@
+# FIWARE Data Model validator
+
+FIWARE Data Model validator is an utility to help the management of NGSI DataModels. Its code leverage on the [AJV JSON Schema Validator](https://github.com/epoberezkin/ajv).
+
+The FIWARE Data Model validator perform the following checks for each Data Model:
+* validity of JSON Schema
+* validity of JSON Examples
+* adherence of Data Model name to FIWARE Data Models guidelines
+* existence of Doc folder
+* existence of JSON Schema
+* existence of one or more JSON examples
+* existence of README file
+
+##  Install the validator
+To install the validator on your machine, you need nodejs 7.0.0+. Instructions on how
+to install nodejs are available [here](https://nodejs.org/en/download/package-manager/).
+
+Once nodejs installed in your system, you can install the validator with the following command:
+
+```
+npm install -g fiware-model-validator
+```
+
+## Using the validator
+To use the validator, execute it from the root of the DataModel repository:
+```
+validate -p DataModel -w ignore -i [common-schema.json,geometry-schema.json]
+```
+
+Command line available options are:
+
+*  ``-i, --fiware:importSchemas``.  Additional schemas that will be included during validation. Default imported schemas are: common-schema.json, geometry-schema.json   [array]
```

Comment:
do we need to prepend the word `fiware` to all the options?

Reply:
replaced with dmv